<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2106</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilt Shifter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="cmdHBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Bar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdVBar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>150</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical Bar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdCircle">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>150</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSaturate">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>136</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>1001</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>1031</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2106</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
